Mathlib/Algebra/TrivSqZeroExt.lean

```diff
@@ -615,20 +615,20 @@ theorem fst_pow [Monoid R] [AddMonoid M] [DistribMulAction R M] [DistribMulActio
 
 theorem snd_pow_eq_sum [Monoid R] [AddMonoid M] [DistribMulAction R M] [DistribMulAction Rᵐᵒᵖ M]
     (x : tsze R M) (n : ℕ) :
-    snd (x ^ n) = ((List.range n).map fun i => x.fst ^ (n.pred - i) • op (x.fst ^ i) • x.snd).sum :=
+    snd (x ^ n) = ((List.range n).map fun i => x.fst ^ (n.pred - i) •> x.snd <• x.fst ^ i).sum :=
```

Comment:
Note that this statement now has different associativity compared to what it was before.

Reply:
Thanks, but I'm not concerned about this change. It's minor, and I think the precedences for the `•>` and `<•` notations are good, so maybe this should be considered the more natural version.

Reply:
I agree that it's minor, which is why I made it anyway :)